test/augmentation/test_augmentation_mix.py

```diff
@@ -387,3 +387,113 @@ def test_p(self, p, same_on_batch, device, dtype):
         input = torch.randn((12, 3, 256, 256), device=device, dtype=dtype)
 
         f(input)
+
+
+class TestRandomTransplantation:
+    @pytest.mark.parametrize("input_3D", [False, True])
```

Comment:
shouldn't we split the 2d and 3d APIs signature? cc @shijianjian

Reply:
We should. Some checkings are different

Reply:
From a user perspective I would prefer to have only one class since the API is the same for higher dimensional input. A separate `RandomTransplantation3D` class would not do anything different under the hood than the 2D version (or 4D, 5D etc.). If I (as a user) would like to support 2D and 3D input, it is more convenient to have the same class for both cases I think.

Reply:
With `AugmentationSequential` there are some differences as then it indeed only works for 2D input. For that reason, I also created a `RandomTransplantation3D` class for 3D input when used together with `AugmentationSequential`. If `RandomTransplantation` is used on its own, it still works for arbitrary spatial dimensions. I also added this information to the docs.